MAINTAINING.md

```diff
@@ -1,11 +1,12 @@
+<!-- @format -->
+
 # Maintaining CodePair
 
 ## Releasing a New Version
 
 ### 1. Update the version number.
 
-- Update `version` in [package.json in `frontend/`](https://github.com/yorkie-team/codepair/blob/1f10dff1d9c253f921ba136c812383fbb292078f/frontend/package.json#L5) and [package.json in `backend/`](https://github.com/yorkie-team/codepair/blob/1f10dff1d9c253f921ba136c812383fbb292078f/backend/package.json#L3).
-- Update `version` in `package-lock.json` in `frontend/` and `backend/`. It will be automatically updated when you run `npm install`.
+- CodePair uses global version management. Update the `version` field only in the root `package.json` file when making a release.
```

Comment:
With global version management, the value of `import.meta.env.PACKAGE_VERSION` is undefined.
This value is injected by `vite-plugin-package-version` in `frontend/vite.config.ts`.

Reply:
I think I was confused. But, I'm not sure how much detail to explain.